SwiftCompilerSources/Sources/Optimizer/InstructionSimplification/SimplifyBuiltin.swift

```diff
@@ -219,3 +250,17 @@ private extension Type {
     return .unknown
   }
 }
+
+private extension Type {
```

Comment:
You can put these APIs directly in Context.swift and remove `Context.getStatic...`

Reply:
Done